cosmic_ray/commands/report.py

```diff
@@ -1,15 +1,23 @@
 def _print_item(item):
-    return [
-        'job ID: {}'.format(item.work_id),
-        'module: {}'.format(item.module_name),
-        'operator: {}'.format(item.operator_name),
-        'occurrence: {}'.format(item.occurrence),
+    result = item.result_data
+    result_type = item.result_type
+    if result_type in ['normal', 'exception']:
+        result_type = result[1][0]
+    ret_val = [
+        'job ID {}:{}:{}'.format(
+            item.work_id,
+            result_type,
+            item.module_name),
         'command: {}'.format(
             ' '.join(item.command)
             if item.command is not None else ''),
-        'result type: {}'.format(item.result_type),
-        'data: {}'.format(item.result_data)
         ]
+    if result_type == 'timeout':
+        ret_val['timeout'] = '{:.3f} sec'.format(result)
+    else:
+        ret_val += result[1][1]
```

Comment:
I'm a bit worried about this line. I can't run CR where I currently am, so I can't verify my concerns. Will `result` always be indexable in this way? I think it will be OK if `result_type` is "normal" or "exception", but not if it's "no-test" (which shouldn't generally happen). 

Also, since you're using `+=`, you're assuming that `result[1][1]` is an iterable object. Will that always be the case? Or should you instead be using `ret_val.append()` here?

I'll try to look at this more closely this evening when I get home.